controller/tests/features/controller.feature

```diff
@@ -0,0 +1,10 @@
+@rhba-7/rhba70-controller
+Feature: RHBA Standalone Controller tests
+
+  Scenario: Test REST API is secure
+    When container is ready
+    Then check that page is served
+      | property             | value                 |
+      | port                 | 8080                  |
+      | path                 | /services/rest/server |
```

Comment:
This path is not correct for Standalone Controller. It should look lika (URL):(PORT)/(controller.context)/management/servers
This path is used for the Kie Server

Reply:
Thanks for catching this. I will fix it.